components/tray/index.css

```diff
@@ -10,70 +10,96 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-@import "../overlay/index.css";
+@import "../commons/overlay-coretokens.css";
 
 .spectrum-Tray {
-  /* Bug: this must be 0ms, not 0 */
-  --spectrum-dialog-confirm-exit-animation-delay: 0ms;
-  --spectrum-tray-margin-top: 64px;
+  /* Placeholder tokens */
+  --spectrum-tray-exit-animation-delay: 0ms;
+  --spectrum-tray-entry-animation-delay: 160ms;
+
+  /* Maximum width for landscape */
+  --spectrum-tray-max-inline-size: 375px;
```

Comment:
Is there a way to soft-code this value?

Reply:
Not that I know of, but it might be worth adding to the tokens doc. This was the previous static value that was used.